lib/node_modules/@stdlib/stats/base/dnanmean/test/test.ndarray.js

```diff
@@ -130,8 +126,7 @@ tape( 'the function supports a negative `stride` parameter', function test( t )
 		NaN   // 0
 	]);
 
-	N = floor( x.length / 2 );
-	v = dnanmean( N, x, -2, 6 );
+	v = dnanmean( 4, x, -2, 6 );
```

Comment:
```suggestion
	v = dnanmean( 5, x, -2, 8 );
```

There was a bug in the previous tests. The offset was incorrect and needs to be updated to reflect what indicated by the commented values. This should be fixed here and elsewhere. Consider the comments `// 0`, etc, to be the intent. The call signatures should match the intent.